dashboard/client/src/pages/serve/ServeSystemActorDetailPage.tsx

```diff
@@ -0,0 +1,179 @@
+import { Typography } from "@material-ui/core";
+import React from "react";
+import { useParams } from "react-router-dom";
+import { CollapsibleSection } from "../../common/CollapsibleSection";
+import { generateActorLink, generateNodeLink } from "../../common/links";
+import {
+  MultiTabLogViewer,
+  MultiTabLogViewerTab,
+} from "../../common/MultiTabLogViewer";
+import { Section } from "../../common/Section";
+import { MetadataSection } from "../../components/MetadataSection";
+import { StatusChip } from "../../components/StatusChip";
+import { ActorDetail } from "../../type/actor";
+import { ServeHttpProxy } from "../../type/serve";
+import { useFetchActor } from "../actor/hook/useActorDetail";
+import { MainNavPageInfo } from "../layout/mainNavContext";
+import { useServeHTTPProxyDetails } from "./hook/useServeApplications";
+
+type ActorInfo = {
+  type: "httpProxy";
+  detail: ServeHttpProxy;
+};
+
+type ServeSystemActorDetailProps = {
+  actor: ActorInfo;
+};
+
+export const ServeHttpProxyDetailPage = () => {
+  const { httpProxyId } = useParams();
+
+  const { httpProxy } = useServeHTTPProxyDetails(httpProxyId);
+
+  if (!httpProxy) {
+    return (
+      <Typography color="error">
+        HTTPProxyActor with id "{httpProxyId}" not found.
+      </Typography>
+    );
+  }
+
+  return (
+    <div>
+      <MainNavPageInfo
+        pageInfo={{
+          id: "serveHttpProxy",
+          title: `HTTPProxyActor:${httpProxy.node_id}`,
+          pageTitle: `${httpProxy.node_id} | Serve HTTPProxyActor`,
+          path: `/serve/httpProxies/${encodeURIComponent(httpProxy.node_id)}`,
+        }}
+      />
+      <ServeSystemActorDetail
+        actor={{ type: "httpProxy", detail: httpProxy }}
+      />
+    </div>
+  );
+};
+
+export const ServeSystemActorDetail = ({
+  actor,
+}: ServeSystemActorDetailProps) => {
+  const name = `HTTPProxyActor:${actor.detail.actor_id}`;
+
+  const { data: fetchedActor } = useFetchActor(actor.detail.actor_id);
+
+  return (
+    <div>
+      <MetadataSection
+        metadataList={[
+          {
```

Comment:
nit: future TODO should be to extract some `createMetadata` and extended `createMetadataWithLink` fns. While this does improve readability, a lot of this ends up being boilerplate